### PR TITLE
fix(theme): css caret

### PR DESCRIPTION
**Pull request by jsomsanith**
**What is the problem this PR is trying to solve?**
Some carets are broken
capture d ecran 2018-07-17 a 10 16 34

**What is the chosen solution to this problem?**
* Reshape caret in theme `.dropdown-toggle caret`
* Remove `dropdown-caret-color` mixin. (Now the caret takes the color of its container)
* Remove `$dropdown-caret-color` variable, that is not used.

* (Bonus: fix button font-weight to fit guideline and fix Toggle storybook)

**Please check if the PR fulfills these requirements**

* [ ] The PR commit message follows our [guidelines](https://github.com/talend/tools/blob/master/tools-root-github/CONTRIBUTING.md)
* [ ] Tests for the changes have been added (for bug fixes / features) And [non reg](./screenshots.md) done before need review
* [ ] Docs have been added / updated (for bug fixes / features)
* [ ] Related design / discussions / pages (not in jira), if any, are all linked or available in the PR

**[ ] This PR introduces a breaking change**

### CODE REVIEW

File: packages/theme/src/theme/_dropdowns.scss [hunk 1]
```diff
@@ -16,10 +16,10 @@ $dropdown-caret-up-axis: -45deg !default;
 			margin: 0 0 $dropdown-caret-size $dropdown-caret-size;
 			height: $dropdown-caret-size;
 			width: $dropdown-caret-size;
-			border-width: 1px !important;
-			border-style: solid;
+			border: 1px solid;
 			border-bottom-color: transparent;
 			border-left-color: transparent;
+			box-shadow: 1px -1px 0px;
```
Comment (jare-talend) [inline packages/theme/src/theme/_dropdowns.scss]: remove 0px by 0

Comment (jare-talend) [reply packages/theme/src/theme/_dropdowns.scss]: no surge demo ?

Comment (jsomsanith) [reply packages/theme/src/theme/_dropdowns.scss]: nope, issue with travis. Waiting for them to fix, for now we removed the demos in #1548

Comment (jsomsanith) [reply packages/theme/src/theme/_dropdowns.scss]: This PR revert the deactivation, it will me merged when travis is green again : #1549

**Pull request merged**